templates/html/.partials/schema-prop.html

```diff
@@ -89,6 +89,9 @@
             {% endfor %}
           </div>
         {% endif %}
+        {% for extensionName, extensionValue in prop.extensions() %}
+            <div class="text-sm markdown">{{ extensionName }}: {{ extensionValue }}</div>
```

Comment:
`extensionValue` can be huge here depending on the extension. I wonder if we should think about another way of visualizing it. Also, if it's a JSON object, doesn't it render `[Object object]`?